### PR TITLE
Validation: Fix multiple issues 

### PR DESCRIPTION
**Pull request by annmariehitchcock**
This PR addresses the following validation issues and includes an updated ms.date: 
missing description 
disallowed html attribute

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[562a49e](https://github.com/annmariehitchcock/sp-dev-docs/commits/562a49e41941694ee0ed4f42f32a5ee97725d905)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/property-element-in-properties-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/property-element-in-properties-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/property-element-in-properties-bdcmetadata-schema?branch=pr-en-us-8103) | [Details](#user-content-8b8313964a37a79838cbe0a24d0d1325-a3c87ec7)

<a id="8b8313964a37a79838cbe0a24d0d1325-a3c87ec7"></a>
### [docs/schema/property-element-in-properties-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/property-element-in-properties-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206092230150090-8103/BuildReport?accessString=e4d0203eadd5702ebbb9469c3440070ca86d5a07af7912848697141513a42f16).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/schema/property-element-in-properties-bdcmetadata-schema.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 title: Property element in Properties (BDCMetadata)
+description: Describes the Property element in Properties (BDCMetadata) and provides the the elements and attributes.
```
Comment (MeeraDi) [inline docs/schema/property-element-in-properties-bdcmetadata-schema.md:3]: Please remove the second "the" as: ...provides the elements and attributes.

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[6f04947](https://github.com/annmariehitchcock/sp-dev-docs/commits/6f049472bf809ca69aa454d6cac5a52a490ab3e1)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/property-element-in-properties-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/property-element-in-properties-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/property-element-in-properties-bdcmetadata-schema?branch=pr-en-us-8103) | [Details](#user-content-8b8313964a37a79838cbe0a24d0d1325-7e0e9024)

<a id="8b8313964a37a79838cbe0a24d0d1325-7e0e9024"></a>
### [docs/schema/property-element-in-properties-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/property-element-in-properties-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206092247219340-8103/BuildReport?accessString=9100d51b73164061d92f694d7646fbc911db5a30e1d042e25d453c22efbfb408).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by annmariehitchcock** [comment]
This PR addresses the following validation issues and includes an updated ms.date: 
missing description 
disallowed html attribute

**Global effort to fix build validation errors**
@VesaJuvonen 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request closed without merging**